src/GovUk.Education.ExploreEducationStatistics.Admin/Models/ImportStatusBau.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using GovUk.Education.ExploreEducationStatistics.Common.Converters;
+using GovUk.Education.ExploreEducationStatistics.Common.Services;
+using GovUk.Education.ExploreEducationStatistics.Data.Processor.Model;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Models
+{
+    public class ImportStatusBau
+    {
+        public Guid SubjectId { get; set; }
+        public string DataFileName { get; set; }
+        public string MetaFileName { get; set; }
+        public Release Release { get; set; }
+        public string Errors { get; set; }
+        public int NumberOfRows { get; set; }
+        public IStatus Status { get; set; }
+        public int StagePercentageComplete { get; set; }
+
+        public override string ToString()
```

Comment:
Don't think this needs `ToString` as it's just a view model and we're not likely to need to log it.